content/rs/administering/database-operations/slave-ha.md

```diff
@@ -58,22 +56,24 @@ You can see the current configuration options for slave HA with: `rladmin info c
 
 ### Grace Period
 
-By default, slave HA has a 15-minute grace period after node failure and before new slave shards are created.
+By default, slave HA has a 10-minute grace period after node failure and before new slave shards are created.
```

Comment:
That's 15 min....

Reply:
@bmansheim I took a look at the code. It was changed as you said to 10 min (from 15)
@Paz-Yanover  FYI